shared/reducers/config.tsx

```diff
@@ -1,5 +1,7 @@
+import {store} from 'emoji-mart'
```

Comment:
i don't think this belongs in the reducer. better as a side effect. also does emoji-mart work on mobile?

Reply:
nope will short circuit on mobile